modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminDisplayContext.java

```diff
@@ -528,6 +528,9 @@ private SearchContainer<RepositoryEntry> _getDLSearchContainer()
 		long fileEntryTypeId = ParamUtil.getLong(
 			_httpServletRequest, "fileEntryTypeId", -1);
 
+		String[] fileExtensions = ParamUtil.getStringValues(
+			_httpServletRequest, "extension");
```

Comment:
These values are referenced both as "extension" or "fileExtension" in different places. Pick one name and use it consistently. Ideally, we should use the same names for variables, req. parameters, methods, etc.

Reply:
You are right, I started with `fileExtension` and at the end I was forced to use the request parameter name `extension`, but I didn't change all related variables and methods.